modules/assim.sequential/R/build_X.R

```diff
@@ -19,7 +19,6 @@
 #' @return X ready to be passed to SDA Analysis code
 #' @export
 #'
-#' @examples
```

Comment:
Is removing @examples necessary to get the documentation to build or something?  CRAN will require examples I believe.  It might be better to put _something_ here wrapped in `\dontrun{}` rather than remove the roxygen tag entirely?

Reply:
This could probably even be something like :

``` r
#' @examples
#' #TODO: add examples!
```

Reply:
Is this an internal function (?) if so could remove `@export` and skip examples?

Reply:
I deleted it because an empty `@examples` triggers complaints from Roxygen, but this was an add-on that could be handled in a different PR. Reverting.

Reply:
@helge22a Do you use `build_X` or `sda_matchparam` anywhere outside `sda.enkf.multisite`? If not I'll make it internal. (I'll handle all the code changes, just need confirmation I'm not breaking your workflows when I do it.)

Reply:
@infotroph these can be internal functions

Reply:
Fixed in 33bfb007b